lib/connect.js

```diff
@@ -689,6 +690,29 @@ regular).
         return _post("order.margins", orders, null, null, true);
     }
 
+    /**
+     * Fetch basket margin for list of orders
+     * @method orderBasketMargins
+     * @memberOf KiteConnect
+     * @instance
+     * @param {Object[]} orders   Margin fetch orders.
+     * @param {string} orders[].exchange    Name of the exchange(eg. NSE, BSE, NFO, CDS, MCX)
+     * @param {string} orders[].tradingsymbol   Trading symbol of the instrument
+     * @param {string} orders[].transaction_type   eg. BUY, SELL
+     * @param {string} orders[].variety   Order variety (regular, amo, bo, co etc.)
+     * @param {string} orders[].product   Margin product to use for the order
+     * @param {string} orders[].order_type   Order type (MARKET, LIMIT etc.)
+     * @param {number} orders[].quantity   Quantity of the order
+     * @param {number} orders[].price   Price at which the order is going to be placed (LIMIT orders)
+     * @param {number} orders[].trigger_price   Trigger price (for SL, SL-M, CO orders)
+     * @param {string} consider_positions    Boolean to consider users positions while calculating margins
+     * @param {string} mode    (optional) Compact mode will only give the total margins
+     */
+    self.orderBasketMargins = function(orders, consider_positions=true, mode=null){
```

Comment:
You might want to enhance `orderMargins` as well to support the extra optional param `mode`?

Reply:
Sure, [added](https://github.com/zerodha/kiteconnectjs/pull/49/commits/b82b0896a7eb4f8fcf0740557212c348ac0c77c1).